prime-router/src/test/kotlin/history/azure/SubmissionFunctionTests.kt

```diff
@@ -387,27 +386,28 @@ class SubmissionFunctionTests : Logging {
         action.actionName = TaskAction.receive
         every { mockSubmissionFacade.fetchActionForReportId(any()) } returns action
         every { mockSubmissionFacade.fetchAction(any()) } returns null // not used for a UUID
-        every { mockSubmissionFacade.findDetailedSubmissionHistory(any(), any()) } returns returnBody
+        every { mockSubmissionFacade.findDetailedSubmissionHistory(any()) } returns returnBody
         every { mockSubmissionFacade.checkSenderAccessAuthorization(any(), any()) } returns true
         response = function.getReportDetailedHistory(mockRequest, goodUuid)
         assertThat(response.status).isEqualTo(HttpStatus.OK)
         var responseBody: DetailSubmissionHistoryResponse = mapper.readValue(response.body.toString())
         assertThat(responseBody.submissionId).isEqualTo(returnBody.actionId)
         assertThat(responseBody.overallStatus).isEqualTo(returnBody.overallStatus.toString())
 
-        // Good uuid, but not a 'receive' step report.
+        // Good uuid, but not a with `process` action step report.
```

Comment:
I think you need to remove 'not a' in this sentence

Reply:
fixed